src/content/docs/guides/dingo/SPO Guides/005-register-pool.md

```
@@ -0,0 +1,311 @@
+NET="--testnet-magic 2"  #preview
+RELAY1_HOST=55.209.117.58
+RELAY1_PORT=6000
+RELAY2_HOST=55.23.123.206
+RELAY2_PORT=6000
+METADATA_URL=https://website.com/preview-pool-metadata.json
+METADATA_HASH=$(cat $HOME/dingo/previewPoolMetaDataHash.txt)
+EOF
+```
+
+> The example above uses IP addresses. If your relay nodes use DNS names instead, use the following options:
+> ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)